guides/ssl-certificates/cloudDNS.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Cloud DNS
+description: Learn how to use cert-manager to set up SSL certificates using Cloud
+DNS for DNS01 challenges.
+---
+
+[cert-manager](https://cert-manager.io/) allows you to enable HTTPS on your
+Coder installation, regardless of whether you're using
+[Let's Encrypt](https://letsencrypt.org/) or you have your own certificate
+authority.
+
+This guide will show you how to install cert-manager v1.0.1 and set up your
+cluster to issue Let's Encrypt certificates for your Coder installation so that
+you can enable HTTPS on your Coder deployment. It will also show you how to
+configure your Coder hostname and dev URLs.
+
+> We recommend reviewing the official cert-manager
+> [documentation](https://cert-manager.io/docs/) if you encounter any issues or
+> if you want info on using a different certificate issuer.
+
+You must have:
+
+- A Kubernetes cluster with internet connectivity
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- A
+  [Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+
+## Step 1: Add cert-manager to your Kubernetes cluster
```

Comment:
If needed, feel free to add context around applying the `cert-manager` manifest.